### ISSUE TITLE
ImportError: cannot import name 'MultiThreadedAugmenter' from 'batchgenerators.dataloading'

### ISSUE THREAD

**Issue by MehriB** [MIC-DKFZ/nnUNet#753]
Hello,

When I use the `nnunet-inference-on-cpu-and-gpu==1.6.6` package for inference on a pre-trained model.
I get the following error:
`ImportError: cannot import name 'MultiThreadedAugmenter' from 'batchgenerators.dataloading'`

Even though the MultiThreadedAugmenter exists inside "dataloading/multi_threaded_augmenter.py".

Could you please help me with that? 
Is it because the batchgenerators installation had a problem?

Thank you for the help,
Mehri

**Comment by MehriB** [comment]
In general, there is a problem with all modules of batchgenerators.
It can not find them.

**Comment by ChibisukeDragon** [comment]
Only one of my six machines has this problem. And I can not solve this problem...

**Comment by ChibisukeDragon** [comment]
Maybe "pip install batchgenerators==0.21" can solve this problem. I dont know.

**Comment by denbonte** [comment]
Hey All,

I started having the same problem recently.
Suspecting it had something to do with the version of some package, I checked the closed issues before starting the debug process to understand which package it was.

@MehriB maybe check this, as @ChibisukeDragon suggested:
https://github.com/MIC-DKFZ/nnUNet/issues/752

Fabian's answer:
> Hi there,
> this should be working with the newest version of nnU-Net from github. If you want to use the pip installer for nnU-Net you should use an older version of batchgenerators: `pip install batchgenerators==0.21`

I've just tried running the inference process before and after installing the 0.21 version of `batchgenerators`, and not it works.

Hope this helps!
